scripts/build.sh

```diff
@@ -409,10 +409,6 @@ do
         env GOOS=$GOOS GOARCH=$GOARCH GOARM=$GOARM go build -ldflags "$LDFLAGS" -o $ARCHIVE_DIR_SOURCE_UI/$GOOS-$GOARCH/kelp.exe
         check_build_result $?
         echo "successful"
-        
-        echo -n "copying over kelp-start.bat file to the windows build ..."
```

Comment:
we should also delete the `gui/windows-bat-file/kelp-start.bat` file in the repo along with this

Reply:
@debnil did we delete `gui/windows-bat-file/kelp-start.bat`?